test/test-utils/helpers/classic-mode-helper.ts

```diff
@@ -119,7 +122,7 @@ export class ClassicModeHelper extends GameManagerHelper {
    * ```
    */
   public async startBattleWithSwitch(pokemonIndex: number): Promise<void> {
```

Comment:
```suggestion
  public async startBattleWithSwitch(pokemonIndex: number): Promise<void> {
    if (!isBetween(pokemonIndex, 1, 5)) {
      expect.fail(`Invalid party index (${pokemonIndex}) passed to \`startBattleWithSwitch\`: must be between 1 and 5 (inclusive)!`);
    }
```
Probably needs the import for `isBetween` added.

I think you can use `type-fest` to make the typing of `pokemonIndex` only 1-5 instead, if you want to look into that.

Reply:
Yup it has an int range type
(Sadly doesn't work on negatives, but i'll live with what i have.)